htdocs/fourn/facture/card.php

```diff
@@ -1004,6 +1004,10 @@
 	            	$array_options,
 	            	$productsupplier->fk_unit
 	            );
+                if ($result<0) {
```

Comment:
All back return of errors should be done into $object->error or $object->errors  ($this->line)
So the fix must be done at a different place, inside the addline. We should trap the error from this->line and push it into this->error or this->errors[]

Reply:
sorry, I finally found my problem